### PR TITLE
Adds JSX.Element support to Slider.label

### PR DESCRIPTION
**Pull request by eamodio**

### CODE REVIEW

**Comment by SergioMorchon** [comment]
Regarding the official documentation, label is of `string` type:
https://dev.fitbit.com/build/reference/settings-api/#slider
> `label` takes a string, the text that appears above the slider.

Anyway, I'll check this with Fitbit stuff to see if they officially support any JSX children on that props, and the same question for #133.

File: types/settings/components.d.ts [hunk 1]
```diff
@@ -31,7 +31,7 @@ declare const Toggle: (props: {
 	onChange?: (newValue: boolean) => void;
 }) => JSX.Element;
 declare const Slider: (props: {
-	label?: string;
+	label?: string | JSX.Element;
```
Comment (SergioMorchon) [inline types/settings/components.d.ts]: `JSX.Element` should be enough, as it inclues `string`. Can you try that? And if it works, just remove the `string` part

Comment (eamodio) [reply types/settings/components.d.ts]: I will try it out and report back

Comment (eamodio) [reply types/settings/components.d.ts]: Yup works fine with just `JSX.Element`

**Pull request merged**